Automattic/wp-calypso
# Update: Redirect subscription only users to the reader

## Proposed Changes

* If a subscrition only user is logs in they are currently redirected to /sites which doesn't help in this case. 
 
## Why are these changes being made?
This change is made so that we provide users that are only subscribers with a more intuitive experience on wordpress.com 



## Testing Instructions

Apply this patch - D150159-code 
Sandbox public-api.wordpress.com 

Load up this patch with a user that only has a subscription. (was created recetly thought the subscription flow) 
Notice when you visit the root of the project. You will be redirected to /read ( the reader ) instead of sites. 

Load up the PR for a regular user. That has a site. 
Notice that you still get redirected as before. 

## Pre-merge Checklist
